### ISSUE TITLE
Draft content for key moments in grandfathering flow

### ISSUE THREAD

**Issue by tygindraux** [department-of-veterans-affairs/va.gov-team#52074]
### Problem statement
Veterans need to make migrate their DS Logon accounts to Login.gov through grandfathering, which allows them to defer identity proofing.

### Initiative
We need to draft content for the grandfathering user flow. This sprint, we'll just focus on moments that are critical and less well-defined by existing prototypes. These are:
- [Asking a Veteran to set up a Login.gov IAL1 account after they sign in to a VA service](https://app.mural.co/t/innovationboards1199/m/innovationboards1199/1671554156732/bb8f2948581f649bf8fd8ad9816224d93818992d?wid=0-1674250041409)
- [Telling a Veteran they've set up a Login.gov (IAL1) account and they should use it moving forward](https://app.mural.co/t/innovationboards1199/m/innovationboards1199/1671554156732/bb8f2948581f649bf8fd8ad9816224d93818992d?wid=11-1674570924974)
- [Telling a Veteran that something went wrong](https://app.mural.co/t/innovationboards1199/m/innovationboards1199/1671554156732/bb8f2948581f649bf8fd8ad9816224d93818992d?wid=6-1674572901288)
- [Telling a Veteran they can upgrade their account to an IAL2 Login.gov account](https://app.mural.co/t/innovationboards1199/m/innovationboards1199/1671554156732/bb8f2948581f649bf8fd8ad9816224d93818992d?wid=4-1674571293449)

#### Collaboration Cycle touchpoint
[Sitewide Content and IA intake request](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request): Submit this request as soon as you have early user flows or designs. This is usually around the time of the Design Intent meeting.

This will be asynchronous, synchronous, or both, depending on needs. The Sitewide Content and IA team offers ongoing collaboration, guidance, and support as you develop your product. OCTO content leads hold regular office hours and can offer product-specific 1:1 collaboration sessions as needed. The team also provides asynchronous content support in tickets as capacity allows.

VFS Product Manager (@elizabethkochBT) submits a Sitewide Content and IA intake form ticket in GitHub. (For reference, our [Collaboration Cycle Request ticket](https://app.zenhub.com/workspaces/logingov-adoption-team-632280b31e745000136a61fc/issues/gh/department-of-veterans-affairs/va.gov-team/50773)).

### Acceptance criteria
- [x] All 4 states have been drafted in Mural
- [x] [Sitewide Content and IA intake request](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) has been scheduled – @CarlDickerson has been advised this is optional and have decided to skip this.

### Tasks
- [x] @CarlDickerson draft content for all 4 states
- [x] @CarlDickerson share with team for review
- [x] @CarlDickerson make any updates based on feedback
- [x] @elizabethkochBT make [Sitewide Content and IA intake request](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) in our [Collaboration Cycle Request ticket](https://app.zenhub.com/workspaces/logingov-adoption-team-632280b31e745000136a61fc/issues/gh/department-of-veterans-affairs/va.gov-team/50773)

**Comment by tygindraux** [comment]
Discussed with @CarlDickerson @MOStudioCarolyn and broke this into tasks. We've agreed to put this on hold until after the Identity Collab Kickoff on Thursday with Oddball, which should provide clarity as to where we should be focused (and avoid us duplicating work they are already doing).

**Comment by tygindraux** [comment]
Discussed with @CarlDickerson @MOStudioCarolyn and broke this into tasks. We've agreed to put this on hold until after the Identity Collab Kickoff on Thursday with Oddball, which should provide clarity as to where we should be focused (and avoid us duplicating work they are already doing).

**Comment by tygindraux** [comment]
@elizabethkochBT I've added you to this ticket as you'll need to make the Sitewide Content and IA intake request, once Carl is ready.

**Comment by tygindraux** [comment]
@CarlDickerson: "I also asked about the Sitewide Content and IA Request - this one is sorta optional for us. It's more designed for teams that don't have a content person for them to setup time to discuss needs etc. But since I'm already working on that content and just need feedback/approval we can workshop in our weekly sync or they can add in a ticket to the epic we already have."

**Comment by tygindraux** [comment]
@CarlDickerson: "I also asked about the Sitewide Content and IA Request - this one is sorta optional for us. It's more designed for teams that don't have a content person for them to setup time to discuss needs etc. But since I'm already working on that content and just need feedback/approval we can workshop in our weekly sync or they can add in a ticket to the epic we already have."
